src/test/isolation2/sql/vacuum_progress_column.sql

```diff
@@ -132,6 +132,9 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
 
 -- Resume execution of compact phase and block at syncrep on one segment.
 2: SELECT gp_inject_fault_infinite('wal_sender_loop', 'suspend', dbid) FROM gp_segment_configuration WHERE role = 'p' and content = 1;
+-- Dispatcher shouldn't proceed until status_version in FtsVersion in FtsProbeInfo is updated,
+-- ensuring the backends are restarted before they start the post-cleanup phase.
+2: SELECT gp_inject_fault('vacuum_rel_finished_one_relation', 'suspend', '', '', 'vacuum_progress_ao_column', 1, 1, 0, 1) FROM master();
```

Comment:
It's not quite clear how moving this fault setup helps in fixing flaky test.

Reply:
The test was flaky because the vacuum managed to clean up indexes before the worker was restarted after FTS version bump. Setting up the fault earlier should help with the edge cases like that.

Another possible solution would be simply removing `heap_blks_scanned` and `heap_blks_vacuumed` columns from the output, since they are mostly irrelevant (the comment mentions num_dead_tuples being reset to 0 after the vacuum worker change, which should be enough to verify the correctness of the test).

Reply:
Still can't understand, how we now make sure that "vacuum managed to clean up indexes before the worker was restarted after FTS version bump" not happens, and how moving fault helps in it.

Reply:
If the fault is set up earlier, it will stop the dispatcher from proceeding (and cleaning up the indexes). `wal_sender_loop` fault was supposed to take care of that, but in rare circumstances it seems like it doesn't. If we move this `vacuum_rel_finished_one_relation` fault, dispatcher definitely can't proceed.

Reply:
okay, thanks